tests/repository_data/targets_simpler.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
correction: "which is not ideal... we should also have separate repositories on the server side"